info.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "RailLogisticsDispatcher",
-  "version": "0.0.11",
+  "version": "0.0.12",
   "title": "Rail Logistics Dispatcher",
-  "author": "Viidi",
+  "author": "Viidi, WristWatch",
```

Comment:
I'll be the sole author to this repository for now :) Please add yourself to the Readme with a very brief description of the contribution. You can also write a note in the changelog, for example, "(by WristWatch)". Hope for understanding

Reply:
Yes, I will do that. Sorry I made a mistake, I was not sure where you wanted me to put my name.